staging/src/k8s.io/client-go/tools/clientcmd/loader.go

```diff
@@ -218,7 +218,8 @@ func (rules *ClientConfigLoadingRules) Load() (*clientcmdapi.Config, error) {
 	}
 
 	if rules.WarnIfAllMissing && len(missingList) > 0 && len(kubeconfigs) == 0 {
-		klog.Warningf("Config not found: %s", strings.Join(missingList, ", "))
+		missingString := strings.Join(missingList, ", ")
+		rules.Warner.Warn(fmt.Sprintf("Config not found: %s", missingString))
```

Comment:
it looks that rules does *not* have Warner method

Reply:
Should I add it there and then make the subsequent changes, I guess changes made here https://github.com/kubernetes/kubernetes/pull/95517/commits/e0b4bce7dea64ab67a654cc9085c5ff50cd4b22d were appropriate

Reply:
e0b4bce looks right to me

Reply:
I have opened a new PR for this #104852 please check that